tests/py_utils/test_utils.py

```diff
@@ -9,6 +9,13 @@ class TestUtils(unittest.TestCase):
     first_duplicate_file_name = 'get_unique_filename_test (1).file'
     second_duplicate_file_name = 'get_unique_filename_test (2).file'
 
+    def test_contains_html(self):
+        html_markup = "<html><p></p></html>"
+        non_html_text = "This is a sentence with special characters, < ? ! . '"
+
+        self.assertEquals(True, utils._contains_html(html_markup))
```

Comment:
line 16 and similar lines show the following problem:

`Cannot access member "assertEquals" for type "TestUtils"
 Member "assertEquals" is unknownPylance [reportGeneralTypeIssues](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportGeneralTypeIssues)`

But the tests run successfully

Reply:
I'm not sure how you're getting this. I don't see it locally and the lint workflow is passing.

Reply:
Must be some issue at my end. I'll merge it.